src/Components/Register.js

```diff
@@ -20,14 +20,19 @@ const Register = () => {
   const handleSubmit = async (e) => {
     e.preventDefault();
     try {
-      const response = await axios.post("http://localhost:8080/api/v1/users/register", input, {
-        headers: {
-          "Content-Type": "application/json",
-        },
-      });
+      const response = await axios.post(
+        "http://localhost:8080/api/v1/users/register",
```

Comment:
Please add the base url in the environment file, so that if the base url is changed we just have to update it over there

Reply:
added the baseURL and .env file for configuration